packages/cli/src/commands/migrate/migrate-command-e2e.spec.ts

```
@@ -0,0 +1,115 @@
+        it('should handle CLI environment variable', () => {
+            const originalEnv = process.env.VENDURE_RUNNING_IN_CLI;
+
+            // Set the environment variable
+            process.env.VENDURE_RUNNING_IN_CLI = 'true';
+            expect(process.env.VENDURE_RUNNING_IN_CLI).toBe('true');
+
+            // Clean up
+            if (originalEnv !== undefined) {
+                process.env.VENDURE_RUNNING_IN_CLI = originalEnv;
+            } else {
+                delete process.env.VENDURE_RUNNING_IN_CLI;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)